### PR TITLE
Feat: Created header, footer and button1 components

### PR DESCRIPTION
**Pull request by Temidayo24**

### CODE REVIEW

File: .vscode/settings.json
Comment (MUFTIATBAKARE) [inline .vscode/settings.json:1]: What does this do?

Comment (Temidayo24) [reply .vscode/settings.json:1]: I didn't put it there neither did i notice.

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-    monospace;
+* {
+  box-sizing: border-box;
+  font-family: "Urbanist";
 }
+
+body {
+  margin: 0;
+  padding: 0;
+  width: 100vw;
```
[the remark this answers is not in the document]
Comment (Temidayo24) [reply src/index.css:10]: I added that for responsiveness. It's a default styling I learnt in the past and I use always now. It's addition or deletion doesn't affect the code in anyway for now so I think it should remain.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "cSpell.words": [
+        "insta"
+    ]
```
Comment (MUFTIATBAKARE) [inline .vscode/settings.json:4]: Please, what does this do?

Comment (Temidayo24) [reply .vscode/settings.json:4]: It's because I added the word insta to my vscode workspace.

**Pull request merged**